src/api.cc

```diff
@@ -55,6 +56,21 @@ AkliteClient::AkliteClient(const std::vector<boost::filesystem::path>& config_di
   client_->finalizeInstall();
 }
 
+void AkliteClient::AssertSingleInstance() {
+  // Leave this open for the remainder of the process to keep the lock held
+  int fd = open("/var/lock/aklite.lock", O_CREAT | O_RDONLY, 0444);
```

Comment:
Is it require the opposite operation - unlocking?

Reply:
no. a flock dies when the file is closed (or process exits).